viz_scripts/scaffolding.py

```diff
@@ -311,29 +349,111 @@ def energy_impact_kWH(df,distance):
     df['Energy_Impact(kWH)']  = round((df['Replaced_mode_EI(kWH)'] - df['Mode_confirm_EI(kWH)']),3)
     return df
 
-def CO2_footprint_lb(df, distance, col):
+def CO2_footprint_default(df, distance, col):
     """ Inputs:
     df = dataframe with data
     distance = distance in miles
     col = Replaced_mode or Mode_confirm
     """
+
+    conversion_lb_to_kilogram = 0.453592 # 1 lb = 0.453592 kg
+    conversion_mile_to_kilometer = 1.60934 # 1 mile = 1.60934 km
+
     conditions_col = [(df[col+'_fuel'] =='gasoline'),
                        (df[col+'_fuel'] == 'diesel'),
                        (df[col+'_fuel'] == 'electric')]
-   
     gasoline_col = (df[distance]*df['ei_'+col]*0.000001)* df['CO2_'+col]
     diesel_col   = (df[distance]*df['ei_'+col]*0.000001)* df['CO2_'+col]
     electric_col = (((df[distance]*df['ei_'+col])+df['ei_trip_'+col])*0.001)*df['CO2_'+col]
 
     values_col = [gasoline_col,diesel_col,electric_col]
     df[col+'_lb_CO2'] = np.select(conditions_col, values_col)
+    df[col+'_kg_CO2'] = df[col+'_lb_CO2'] * conversion_lb_to_kilogram
     return df
     
-  
-def CO2_impact_lb(df,distance):
+def CO2_impact_default(df,distance):
     if 'Mode_confirm_lb_CO2' not in df.columns:
         print("Mode confirm footprint not found, computing before impact")
-        df = CO2_footprint_lb(df, distance, "Mode_confirm")
-    df = CO2_footprint_lb(df, distance, "Replaced_mode")
-    df['CO2_Impact(lb)']  = round((df['Replaced_mode_lb_CO2'] - df['Mode_confirm_lb_CO2']),3)
+        df = CO2_footprint_default(df, distance, "Mode_confirm")
+    df = CO2_footprint_default(df, distance, "Replaced_mode")
+    df['CO2_Impact(lb)'] = round((df['Replaced_mode_lb_CO2'] - df['Mode_confirm_lb_CO2']), 3)
+
+    # Convert the CO2_Impact to be represented in kilogram
+    df['CO2_Impact(kg)']  = round((df['Replaced_mode_kg_CO2'] - df['Mode_confirm_kg_CO2']), 3)
     return df
+
+def compute_CO2_footprint_dynamic(expanded_ct, dynamic_labels, label_type):
+    conversion_meter_to_kilometer = 0.001
+    conversion_kilogram_to_lbs = 2.20462
+    conversion_kilometer_to_mile = 0.621371
```

Comment:
Ditto

Reply:
We're using conversion of `distance`  that is by default `meter` to `kilometer` to compute with `kgCO2perKm` from dynamic config, Similarly, with the above justification, we might want to represent CO2 emission in `Metric` system even when `default mapping` is available. In those case, `conversion_kilogram_to_lbs` will be useful. 
Removed `conversion_kilometer_to_mile` defined above.